src/main/java/com/adyen/Client.java

```diff
@@ -30,43 +30,14 @@
 public class Client {
     private ClientInterface httpClient;
     private Config config;
-    public static final String ENDPOINT_TEST = "https://pal-test.adyen.com";
-    public static final String ENDPOINT_LIVE = "https://pal-live.adyen.com";
     public static final String ENDPOINT_CERT_LIVE = "https://palcert-live.adyen.com";
-    public static final String ENDPOINT_LIVE_SUFFIX = "-pal-live.adyenpayments.com";
-    public static final String API_VERSION = "v68";
     public static final String PAYOUT_API_VERSION = "v68";
-    public static final String RECURRING_API_VERSION = "v68";
     public static final String LIB_NAME = "adyen-java-api-library";
     public static final String LIB_VERSION = "20.1.1";
     public static final String CHECKOUT_ENDPOINT_TEST = "https://checkout-test.adyen.com/checkout";
```

Comment:
Ditto.

Reply:
This one is removed already `RECURRING_API_VERSION `